src/EnergyPlus/SystemAvailabilityManager.cc

```diff
@@ -2167,6 +2167,12 @@ namespace SystemAvailabilityManager {
 		static Array1D_bool ZoneCompNCControlType;
 		int CyclingRunTimeControlType; 
 
+		// reset start/stop times at beginning of each day during warmup to prevent non-convergence due to rotating start times
+		if ( WarmupFlag && BeginDayFlag ) {
```

Comment:
So what does this do, just do it once?

Reply:
It will reset once for each warmup day.  Keeps the start time from moving and messing up the convergence checks.

Reply:
Finished review of code changes. The only question I have is who is king when controlling the fan? Avail manager or load?

Reply:
I'm just trying to think through what the avail managers should do. Lets take this one for example.

    AvailabilityManager:ScheduledOn

If I schedule that fan on should it turn on regardless of load?

If avail manager says to turn on fan, then the fan turns on. And if told to turn off, who cares what the load is ... turn off. I don't think any other model overrides the avail managers.

Reply:
I just looked at the title again. Thermostatic fan control ... so where is that choice? How do I choose Tstat fan control or ? I don't see any IDD changes.